internal/repositories/sql/user/user.go

```diff
@@ -26,7 +26,13 @@ func New(logger *zap.Logger, db *database.SqlDB) user.User {
 }
 
 func (i *impl) CreateUser(req *user.CreateUserRequest) (*user.CreateUserResponse, error) {
-	err := i.db.Master().Create(req.User).Error
+	id, err := nanoid.GetID()
+	if err != nil {
+		return nil, fmt.Errorf("failed to generate nanoid: %v", err)
+	}
+	req.User.UserID = id
```

Comment:
As per signup flow, we do not create new user direclty in users table.
First we create a unverified user in user_verifications and after verification we create a new entry in users table with same user_id as that in user_verifications table.